types/react-native/index.d.ts

```diff
@@ -7288,13 +7288,36 @@ export interface PushNotificationIOSStatic {
      *
      * The type MUST be 'notification'
```

Comment:
This comment (starting with "remote notifications") should probably be updated to include local notifications or just dropped.

Reply:
@Pajn if you check https://facebook.github.io/react-native/docs/pushnotificationios.html#addeventlistener you see I copied (and slightly modified to fit format) their comments. Official documentation says "remote".

Reply:
We can just drop it. a lot of docs in react-native website are outdated.

Reply:
I see. I still think the `The type MUST be 'notification'` should either match the type or be dropped though.

I shall not block the merge though. I will see if I can make @typescript-bot happy again